Documentation/Mobiles For G6.md

```diff
@@ -0,0 +1,141 @@
+
+
+Phone Model | Android Version | Dexcom G5 | Dexcom G6 | Xdrip Version  | Debug Modes | Any Known Issues
+--- | --- | ---  | ---  | ---   | ---  | ---
+Samsung A3 2017 | 6.0.1 |Yes | not known | not known| OB1 | none known
+Samsung Galaxy S4 (SCH-I545) | 5.0.1
+Samsung Galaxy S5 (SM-G900V) | 6.0.1
+Samsung Galaxy Note 5 (N920V) | 7.0
+Samsung Galaxy Note 8 (SM-N950F) | 9 | Yes| Yes|[Nightly build 30th May 2019](https://github.com/NightscoutFoundation/xDrip/releases/tag/2019.05.30)| OB1 - Native Algorithm | none known
+Samsung Galaxy Note 8 (SM-N950F) | 8.0 | Yes| Yes|[Nightly build 2nd Feb 2018](https://github.com/NightscoutFoundation/xDrip/releases/tag/2018.02.02)| OB1 - Native Algorithm | none known
+Samsung Galaxy Note 9 (SM-N960F) | 9 | not known| Yes|[Nightly build 30th May 2019](https://github.com/NightscoutFoundation/xDrip/releases/tag/2019.05.30)| OB1 - Native Algorithm | none known
+Samsung Galaxy S6 (G920V) | 6.0.1
+Samsung Galaxy S7 (SM-G930P) | 7.0
+Samsung Galaxy S7 (SM-G930V) | Android 8 | Yes | Yes | 
+Samsung Galaxy S7 (SM-G930W8) | 6.0.1
+Samsung Galaxy S7 edge (SM-G935V) | 7.0
+Samsung Galaxy S7 edge (G935T) | 7.0
+Samsung Galaxy S9 (SM-G960F) | Android9 | Yes| Yes|[Nightly build 30th May 2019](https://github.com/NightscoutFoundation/xDrip/releases/tag/2019.05.30)| OB1 - Native Algorithm|none known
+Samsung Galaxy S9+ (SM-G965F) | 9 | not known| Yes|[Nightly build 30th May 2019](https://github.com/NightscoutFoundation/xDrip/releases/tag/2019.05.30)| OB1 - Native Algorithm|none known
+Samsung Galaxy Express Prime (SM-J320A) | 6.0.1
+Sony Xperia Z3 Compact (D5803) | 6.0.1 | Yes| Not known |[Nightly build 3rd Jan 2019](https://github.com/NightscoutFoundation/xDrip/releases/tag/2019.05.18)| OB1 - Native Algorithm|Allow ob1 unbonding and initiate bonding checked
+ 
+
+
+Phone Model | Android Version | Dexcom G5 | Dexcom G6 | Xdrip Version  | Debug Modes | Any Known Issues
+--- | --- | ---  | ---  | ---   | ---  | ---
+Xiaomi Mi Mix 2 | Lineage OS 15.1 | Yes  | not known | not known | OB1 | None known
+XiaomiPocophonef1|Android9|notknown|Yes|[Nightly build 21st May 2019](https://github.com/NightscoutFoundation/xDrip/releases/tag/2019.05.21/)|OB1|None known
+
+
+
+Phone Model | Android Version | Dexcom G5 | Dexcom G6 | Xdrip Version  | Debug Modes | Any Known Issues
+--- | --- | ---  | ---  | ---   | ---  | ---
+ZTE Axon 7 Mini | 7.1.1 | Yes  | not known | not known | OB1
+ZTE Axon 7 | 7.1.2 | Yes  | not known | not known | OB1  <br>
+<br>
+
+* * *
+
+
+   
+## Tips
+<br>
+On Doogee / MTK phones disable Intelligent power saving standby in Android Settings -> Battery or you will only receive a reading every 10 minutes.<br>
+* * *
+
+## Any question's you might have or want to share your mobile settings with us that work with Xdrip please free to get in touch!<br>
+<br>
+
+[Contact US](https://www.thediabeticway.co.uk/index.php/en/about-us/contact-us.html) or Visit us on [Facebook](https://www.facebook.com/groups/xDripG5/)
```

Comment:
What is this for?  I don't think this website is related to xDrip+.

Reply:
Its a logo I created if it is needed to be used! I did it for a few videos I made. not important but if you care to want to use it you can! The web site is related to xdrip has it mine!